src/DIRAC/DataManagementSystem/Agent/FTS3Agent.py

```diff
@@ -98,14 +93,16 @@ def __readConf(self):
         # lifetime of the proxy we download to delegate to FTS
         self.proxyLifetime = self.am_getOption("ProxyLifetime", PROXY_LIFETIME)
 
+        # Find out if send to Accounting and/or Monitoring
+        self.monitoringOption = opHelper().getValue("Something/SomethingElse")
```

Comment:
I would say
```suggestion
        self.monitoringOption = opHelper().getValue("DataManagement/MonitoringBackends")
```

and I would actually use 
```suggestion
        self.monitoringOptions = opHelper().getValue("DataManagement/MonitoringBackends", "Accounting").replace(" ", "").split(",")
```
To make sure that `self.monitoringOptions` is a list of strings without spaces, and `["Accounting"]` is default.